dw-text-editor.js

```diff
@@ -56,6 +56,16 @@ class DwTextEditor extends LitElement {
           -webkit-flex-wrap: wrap;
           flex-wrap: wrap;
           border-bottom: 1px solid var(--toolbar-border-color, black);
+          position: sticky;
+          position: -webkit-sticky;
```

Comment:
It seems almost all browsers supports `position: sticky` see https://caniuse.com/css-sticky why do we mention browser specific value then?